content/hi/event-streaming.md

```diff
@@ -0,0 +1,39 @@
+---
+title: इवेंट स्ट्रीमिंग (Event Streaming)
+status: Completed
+category: अवधारणा
+tags: ["क्रियाविधि", "नेटवर्किंग", ""]
+---
+
+## यह क्या है
+
+इवेंट स्ट्रीमिंग एक ऐसा दृष्टिकोण है जहां सॉफ़्टवेयर इवेंट डेटा को एक एप्लिकेशन से दूसरे एप्लिकेशन में भेजता है ताकि वे लगातार संवाद कर सकें कि वे क्या कर रहे हैं।
+कल्पना करें कि एक सेवा वह सब कुछ प्रसारित करती है जो वह अन्य सभी सेवाओं के लिए करता है। किसी सेवा द्वारा की गई प्रत्येक गतिविधि को एक ईवेंट के रूप में संदर्भित किया जाता है, इसलिए इवेंट स्ट्रीमिंग। उदाहरण के लिए, NASDAQ को हर सेकंड स्टॉक और कमोडिटी प्राइसिंग पर अपडेट मिलता है। यदि आपके पास एक ऐसा एप्लिकेशन है जो स्टॉक के एक विशिष्ट सेट की निगरानी करता है, तो आप लगभग रीयल-टाइम में वह जानकारी प्राप्त करना चाहेंगे। 
+याहू! वित्त एक [एपीआई](/application-programming-interface/) प्रदान करता है जो NASDAQ से खींचता है और उनके आवेदन से सूचना (या घटनाओं) को भेजता है (या स्ट्रीम करता है) किसी भी आवेदन के लिए जो इसकी सदस्यता लेता है। भेजे जा रहे डेटा के साथ-साथ उस डेटा में परिवर्तन (स्टॉक की कीमतें) घटनाएँ हैं जबकि उन्हें किसी एप्लिकेशन तक पहुँचाने की प्रक्रिया इवेंट स्ट्रीमिंग है।
+
+
+## समस्या
+
+परंपरागत रूप से, याहू! वित्त एकल टीसीपी अनुरोधों का उपयोग करेगा। यह बहुत अक्षम होगा क्योंकि इसमें प्रत्येक घटना के लिए एक कनेक्शन बनाने की आवश्यकता होगी। 
```

Comment:
english content: `Traditionally, Yahoo! Finance would use single TCP requests. This would be very inefficient as it would require a connection to be created for every event.` uses words like `would use` which may cause the sentence appear to refer to future tense. 

However, it talks about what Yahoo Finance did in the past. please correct this accordingly.

Also use TCP for the protocol since that's standard